alinka/db/connection.py

```diff
@@ -23,9 +22,3 @@
 
 engine = create_engine(f"sqlite:///{settings.DB_PATH}")
 db_session = scoped_session(sessionmaker(bind=engine))
-
-qt_db = QSqlDatabase.addDatabase("QSQLITE")
-qt_db.setDatabaseName(settings.DB_PATH)
-
-# Add handling opening issues https://github.com/CodeForPoznan/alinka-pyside/issues/34
-qt_db.open()
```

Comment:
This has to be moved after application is created, so I've just moved it to `run.py` (I haven't touched SQAlchemy vs QtSql though yet.